create-embeddings/README.md

```diff
@@ -0,0 +1,15 @@
+# Create Embeddings
+
+This folder contains Jupyter Notebooks that describe how to generate
+vector embeddings.
+
+Select one of the following notebooks based on your preferred 
+embedding model, and whether you're generating embeddings from 
+new data or from data you already have in MongoDB Atlas. 
+
+| Notebook | Description |
+|----------|-------------|
```

Comment:
Please check this after you merge - I can't remember if GitHub-flavored Markdown handles cases where the underline characters and header columns aren't as wide as the contents of the column, or if you'll need to pad the columns with spaces/extra underlines to be the same uniform number of characters for the table to display properly.